docs: percentile in usage analytics (#53078)
metabase/metabase: docs/usage-and-performance-tools/usage-analytics.md

- Dashboards with more questions in the same tab
- Users consuming the most resources

> > If you're using MySQL or MariaDB as your application database, you won't be able to see the results for the cards displaying 50th and 90th percentile query running time on the Performance overview dashboard, because MySQL and MariaDB don't support the [Percentile aggregation](../questions/query-builder/expressions-list.md#percentile). We recommend using PostgreSQL as your application database.

Reviewer: ```suggestion
> If you're using MySQL or MariaDB as your application database, the Performance overview dashboard won't display results for the cards displaying the 50th and 90th percentile query running times, because MySQL and MariaDB don't support the [Percentile aggregation](../questions/query-builder/expressions-list.md#percentile). We recommend using PostgreSQL as your application database.
```
